app/views/planning_application/permitted_development_rights/_immunity_details.erb

```diff
@@ -7,17 +7,32 @@
     <strong>Note: application may be immune from enforcement</strong>
   </p>
 
-  <h3 class="govuk-heading-s govuk-!-margin-top-5 govuk-!-margin-bottom-1">Applicant information</h3>
+  <%= render(PlanningApplications::ImmunityDetailsComponent.new(immunity_details: @planning_application.immune_proposal_details)) %>
+
+  <h3 class="govuk-heading-s govuk-!-margin-top-5 govuk-!-margin-bottom-1">Assessment summary — Evidence of immunity</h3>
   <ul class="govuk-body immunity-details-sub-list">
-    <% @planning_application.immune_proposal_details.each do |proposal_detail| %>
-      <li>
-        <%= proposal_detail.question %> 
-        <% proposal_detail.response_values.each do |response_value| %>
-          <strong>
-            <%= response_value.to_time.nil? ? response_value : response_value.to_time.to_formatted_s(:day_month_year_slashes) %>
-          </strong>
+    <li><strong>Evidence cover:</strong>
+      <% if  planning_application.immunity_detail.start_date.present? %>
+        <%= planning_application.immunity_detail.start_date.to_fs(:day_month_year_slashes) %>
+        <% if  planning_application.immunity_detail.end_date.present? %>
+          to <%= planning_application.immunity_detail.end_date.to_fs(:day_month_year_slashes) %>
```

Comment:
Sorry, I think my last comment about end dates might have confused this. I think we still need to find the latest date that the evidence covers. The end date on the immunity detail is when the applicant claims that the work was finished.

Reply:
Oh I see, yeah — it's that the names conflict, not that the method isn't needed. (Now I realise why you'd used different names.)